src/Mvc/test/WebSites/GenericHostWebSite/GenericHostWebSite.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IsTestAssetProject>true</IsTestAssetProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Common\TestResponseGenerator.cs" Link="TestResponseGenerator.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Mvc" />
+    <Reference Include="Microsoft.AspNetCore.Mvc.Formatters.Xml" />
+    <Reference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" />
+
+    <Reference Include="Microsoft.AspNetCore.Localization.Routing" />
+
+    <!--
+      Explicitly referencing routing here to make it easy to integrate breaking changes. Localization.Routing brings in a transitive
```

Comment:
??? These are all resolved to project references. I don't think these NoWarns should be necessary anymore. If they are, I'd like to inspect the scenarios which lead to this. NoWarn may be hiding other problems.

Reply:
Removed.

Reply:
Are you sure? I still see it in the diff.